metricbeat/module/docker/config.go

```diff
@@ -0,0 +1,22 @@
+package docker
+
+type TlsConfig struct {
+	Enable   bool   `config:"enable"`
```

Comment:
We currently use `enabled` as standard (additional d at the end). We had both in beats in the past and picked `enabled`.